.github/test-api-use/contracts/API-Inclusion.sol

```diff
@@ -0,0 +1,73 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.18;
+
+import {FilAddresses} from "filecoin-solidity-api/contracts/v0.8/utils/FilAddresses.sol";
+import {CommonTypes} from "filecoin-solidity-api/contracts/v0.8/types/CommonTypes.sol";
+
+import {AccountTypes} from "filecoin-solidity-api/contracts/v0.8/types/AccountTypes.sol";
+import {DataCapTypes} from "filecoin-solidity-api/contracts/v0.8/types/DataCapTypes.sol";
+import {MarketTypes} from "filecoin-solidity-api/contracts/v0.8/types/MarketTypes.sol";
+import {MinerTypes} from "filecoin-solidity-api/contracts/v0.8/types/MinerTypes.sol";
+import {PowerTypes} from "filecoin-solidity-api/contracts/v0.8/types/PowerTypes.sol";
+import {VerifRegTypes} from "filecoin-solidity-api/contracts/v0.8/types/VerifRegTypes.sol";
+
+import {AccountAPI} from "filecoin-solidity-api/contracts/v0.8/AccountAPI.sol";
+import {DataCapAPI} from "filecoin-solidity-api/contracts/v0.8/DataCapAPI.sol";
+import {MarketAPI} from "filecoin-solidity-api/contracts/v0.8/MarketAPI.sol";
+import {MinerAPI} from "filecoin-solidity-api/contracts/v0.8/MinerAPI.sol";
+import {PowerAPI} from "filecoin-solidity-api/contracts/v0.8/PowerAPI.sol";
+import {PrecompilesAPI} from "filecoin-solidity-api/contracts/v0.8/PrecompilesAPI.sol";
+import {SendAPI} from "filecoin-solidity-api/contracts/v0.8/SendAPI.sol";
+import {VerifRegAPI} from "filecoin-solidity-api/contracts/v0.8/VerifRegAPI.sol";
+
+contract API_Inclusion {
+    //AccountAPI
+    function account_authenticate_message(CommonTypes.FilActorId target, AccountTypes.AuthenticateMessageParams memory params) public view {
+        AccountAPI.authenticateMessage(target, params);
+    }
+
+    //AddressAPI
+    function actorid_conversion() public pure {
+        uint64 actorID = 1;
+        CommonTypes.FilAddress memory result = FilAddresses.fromActorID(actorID);
+    }
+    //DatacapAPI
+    function bundle(DataCapTypes.TransferParams memory params) internal returns (int256, DataCapTypes.TransferReturn memory) {
```

Comment:
What is the expectation here related to code coverage? There are two DataCapAPI calls to name and symbol, but there are many other functions in the [DataCapAPI Library](https://github.com/filecoin-project/filecoin-solidity/blob/master/contracts/v0.8/DataCapAPI.sol). Do we need to have a full test here? Is the intention to show that the api is included and therefore a full comprehensive test is not needed here? Can the status be documented in a comment in this file?

Reply:
This is a basic install and compile test after publishing new package version. It's just one contract that imports all *APIs and uses one function from each - meant to resemble the way lib user would interact with the lib.

`> Can the status be documented in a comment in this file?`
Not sure about 'status' means here?

Reply:
No ok that makes sense - no need for any changes here - thank you for clarifying